### PR TITLE
All pagination to continue without state.

### PR DESCRIPTION
**Pull request by BrydenOliver**
Allow stateless servers to return a continuation token and then use the token to continue the iteration.

### CODE REVIEW

**Comment by bhelx** [comment]
@BrydenOliver Can you further explain how the current code doesn't meet your expectations? I'm happy to review and merge this but I want to fully understand the problem.

**Comment by BrydenOliver** [comment]
Ok, so we are using Recurly behind our own web service. So what happens is a request comes into our web service for the first page (200 items) of our list, so we call Accounts.List() and then retrieve the first page from Recurly. We then want to return to our caller with a continuationtoken so they can continue from where they finished.
There is a continuationtoken returned by Recurly, the client library just hides it right now.
So when our caller then wants the next page we have no continuationtoken or any method to continue the iteration. That's what the changes do, just expose that again.

**Comment by BrydenOliver** [comment]
@bhelx Explanation above. Thanks.

File: Library/List/AccountList.cs [hunk 1]
```diff
@@ -23,6 +23,11 @@ internal override void ReadXml(XmlTextReader reader)
 
         }
 
+        public static AccountList Continue(string continuationToken)
+        {
+            return new AccountList(continuationToken);
```
[the remark this answers is not in the document]
Comment (BrydenOliver) [reply Library/List/AccountList.cs]: There probably are, I don't know the Client Library code base well enough, but this pattern should be pretty consistent. You could take a look at the otehr uses of RecurlyList and see if this makes sense there too.

**Comment by aaron-suarez** [comment]
@BrydenOliver I have confirmed that all uses of RecurlyList indeed use pagination with a similar token. You can read more about it in the [pagination documentation](https://dev.recurly.com/docs/pagination).

If you want to go ahead and edit this PR to abstract the token getter out to RecurlyList, we can go ahead and get this reviewed and merged.

**Comment by aaron-suarez** [comment]
Closing this out due to inactivity. Please feel free to re-open with the continue method in RecurlyList instead of AccountList.

**Pull request closed without merging**